site/src/pages/GitAuthPage/GitAuthPageView.tsx

```diff
@@ -76,7 +76,7 @@ const GitAuthPageView: FC<GitAuthPageViewProps> = ({
     <SignInLayout>
       <Welcome message={`You've authenticated with ${gitAuth.type}!`} />
       <p className={styles.text}>
-        Hey @{gitAuth.user?.login} 👋!{" "}
+        Hey @{gitAuth.user?.login}! 👋{" "}
```

Comment:
Nitpick, but should there be formatting for when the login isn't available for whatever reason? If `gitAuth.user?.login` evaluates to `undefined`, then the user would see `Hey @! 👋`. Maybe a generic `Hey there!` would be good?

Reply:
I just don't wanna creep the scope of this PR too much with things that _might_ be wrong. I was just correcting something that _definitely_ looked wrong to me. 😅 idk if that seems like a silly difference, but if I chased every weird thing I noticed I'd never finish any PRs.

Reply:
No, that's fair, and that's a perspective I really should adopt more myself